src/shared/Microsoft.AzureRepos.Tests/AzureReposHostProviderTests.cs

```diff
@@ -171,5 +171,88 @@ public async Task AzureReposProvider_GetCredentialAsync_ReturnsCredential()
             Assert.Equal(personalAccessToken, credential.Password);
             // We don't care about the username value
         }
+
+        [Fact]
+        public async Task AzureReposHostProvider_ConfigureAsync_UseHttpPathSetTrue_DoesNothing()
+        {
+            string key = $"{Constants.GitConfiguration.Credential.SectionName}.https://dev.azure.com.{Constants.GitConfiguration.Credential.UseHttpPath}";
+            var provider = new AzureReposHostProvider(new TestCommandContext());
+
+            var environment = new TestEnvironment();
+            var config = new TestGitConfiguration(new Dictionary<string, IList<string>>
+            {
+                [key] = new List<string> {"true"}
+            });
+
+            await provider.ConfigureAsync(
+                environment, EnvironmentVariableTarget.User,
+                config, GitConfigurationLevel.Global);
+
+            Assert.Single(config.Dictionary);
+            Assert.True(config.Dictionary.TryGetValue(key, out IList<string> actualValues));
+            Assert.Single(actualValues);
+            Assert.Equal("true", actualValues[0]);
+        }
+
+        [Fact]
+        public async Task AzureReposHostProvider_ConfigureAsync_UseHttpPathSetFalse_SetsUseHttpPathTrue()
+        {
+            string key = $"{Constants.GitConfiguration.Credential.SectionName}.https://dev.azure.com.{Constants.GitConfiguration.Credential.UseHttpPath}";
```

Comment:
nit: this string is used in several places, maybe a public const somewhere instead